pycloudlib/ibm/cloud.py

```diff
@@ -293,6 +323,7 @@ def launch(
             vpc: VPC to allocate the instance in. If not given, the instance
             username: username to use when connecting via SSH
             will be allocated in the zone's default VPC.
+            existing_floating_ip_name_includes: string, name to filter floating IPs by
```

Comment:
Could document here that `if using an existing floating ip, it won't be deleted`? Feel free to rephrase it please.

Reply:
good idea. i modified the docstring explanation. let me know if you think this explanation is not sufficient